src/mumble/MainWindow.cpp

```diff
@@ -1615,6 +1616,12 @@ void MainWindow::qmUser_aboutToShow() {
 	if (g.s.bTTS)
 		qmUser->addAction(qaUserLocalIgnoreTTS);
 	qmUser->addAction(qaUserLocalVolume);
+
+	if (p) {
+		qaUserLocalVolumeSlider = new UserLocalVolumeSlider(p->uiSession);
+		qmUser->addAction(qaUserLocalVolumeSlider);
+	}
```

Comment:
A couple of things:
1. Do we really have to create this action anew every time? This seems like something we could store in a map on a per-user basis
2. Did you verify that the QMenu actually takes ownership of the passed action? If not, this would be a memory leak
Alternatively the actions could be made in such a way that they can easily switch out the user/channel they are acting on. In that case we would create the action once and then only update the current channel/user as needed. That'd probably be the best method :thinking:

Reply:
I was really unsure of this one and decided to go with the more compact solution.

I also considered doing it the way you mention in the second point but I would have to deal with QMenu deleting the action.

I was lead to believe QMenu takes ownership by addAction but after a lot of searching, it seems this doesn't happen when passing a QAction. 

I will try to implement as described in the second point.

Reply:
> I was lead to believe QMenu takes ownership by addAction but after a lot of searching, it seems this doesn't happen when passing a QAction.

Yeah that was what I found as well. I think chances are that it also takes ownership of a `QAction *` just as it does for all other overloads of `addAction` but the docs never state so explicitly. Thus I was asking if you have found any other documentation source or tested it out...

> I will try to implement as described in the second point.

:ok_hand: